docs/architecture/decisions/0019-translation-process-for-english-welsh-strings.md

```diff
@@ -0,0 +1,110 @@
+
+Currently, the translation process is manual. To move toward a structured and partially automated workflow, we need to define how untranslated or updated strings are sent to translators and how completed translations are brought back into the codebase.
+
+We also need to ensure consistency between English and Welsh locale files by validating that both exist and contain matching keys.
+
+### Existing Translation Checks
+
+Two scripts are already in place to validate translation completeness:
+
+1. **Locale File Existence Check**
+   - Verifies that if an `en.json` file exists in a folder, a corresponding `cy.json` must also exist.
+   - Produces a list of missing locale files (i.e., folders where one of the language files is missing).
+
+2. **Key Consistency Check**
+   - Compares keys between `en.json` and `cy.json`.
+   - Lists missing keys in either file and identifies discrepancies.
+
+These checks ensure both languages have parallel structures and help identify untranslated or missing strings.
+
+### Translation Format and Delivery
+
+The translation team currently accepts **Excel** files as standard, with the possibility of also accepting **JSON** files.  
+Data can be sent via email or uploaded to a platform (to be confirmed).
+
+---
+
+## Decision
+
+### Chosen Approach
+
+We will adopt a flexible export/import process that supports both Excel and JSON formats, defaulting to Excel unless a translation platform can accept JSON uploads.
+
+This approach allows us to align with the translation team’s workflow while preparing for future automation if a platform is introduced.
+
+### Excel Structure
+
+Each exported Excel file will include four columns:
+
+| Namespace               | Key             | en                         | cy                       |
+| ----------------------- | --------------- | -------------------------- | ------------------------ |
+| Folder name (namespace) | Translation key | English string (if exists) | Welsh string (if exists) |
+
+This format provides the translators with clear context for each string while retaining the mapping required to import translations back into JSON files.
+
+---
+
+## Consequences
+
+### Export Process
+
+A script will generate an export file (Excel by default, JSON if required) containing all untranslated or updated strings.
+
+The export script will:
+
+- Identify missing or updated strings using the existing comparison scripts.
+- Compile them into a structured dataset.
+- Write the dataset into an Excel file (e.g. `translations-export.xlsx`).
+
+### Translation Process
+
+The translation team will complete the Welsh or English (`cy` | `en`) column and return the file via email or upload.  
+If JSON format is supported, they may instead provide updated `cy.json` files directly depending on their setup.
```

Comment:
Would we expect the translation team to handle multiple translation files in this instance?